src/components/tasks/card/index.tsx

```diff
@@ -29,42 +24,56 @@ const Card: FC<Props> = ({ content }) => {
 
   const localStartedOn = new Date(parseInt(startedOn, 10) * 1000);
   const fromNowStartedOn = moment(localStartedOn).fromNow();
-  const readableStartedOn = `${localStartedOn.toLocaleDateString()}, ${localStartedOn.toLocaleTimeString()}`;
 
   const localEndsOn = new Date(parseInt(endsOn, 10) * 1000);
   const fromNowEndsOn = moment(localEndsOn).fromNow();
-  const readableEndsOn = `${localEndsOn.toLocaleDateString()}, ${localEndsOn.toLocaleTimeString()}`;
+
+  const statusFontColor = status === 'active' || 'assigned' || 'completed' || 'pending' ? '#00a337' : '#f83535';
 
   return (
     <div className={classNames.card}>
-      <span className={classNames.prTitle}>{title}</span>
-      <div className={classNames.datetime}>
-        <span className={classNames.nothover}>
-          {informationElement('Estimated completion', fromNowEndsOn)}
+      <div className={classNames.cardItems}>
+        <span className={classNames.cardTitle}>{title}</span>
+        <span>
+          <span className={classNames.cardSpecialFont}>Status:</span>
+          <span className={classNames.cardStatusFont} style={{ color: statusFontColor }}>
+            {status}
+          </span>
         </span>
-        <span className={classNames.onhover}>
-          {informationElement('Estimated completion', readableEndsOn)}
+      </div>
+      <div className={classNames.cardItems}>
+        <span>
+          <Image
+            src="/calendar-icon.png"
+            alt="calendar icon"
+            width="25px"
+            height="25px"
+          />
+          <span className={classNames.cardSpecialFont}>
+            Due Date
+          </span>
+          <span className={classNames.cardStrongFont}>{fromNowEndsOn}</span>
         </span>
       </div>
-      <div className={classNames.datetime}>
-        <span className={classNames.nothover}>
-          {informationElement('Started', fromNowStartedOn)}
+      <div className={classNames.cardItems}>
+        <span className={classNames.cardSpecialFont}>
+          Started
+          {' '}
+          {fromNowStartedOn}
         </span>
-        <span className={classNames.onhover}>
-          {informationElement('Started', readableStartedOn)}
+        <span>
+          <span className={classNames.cardSpecialFont}>Assignee:</span>
+          <span className={classNames.cardStrongFont}>{assignee}</span>
+          <span>
+            <img
+              className={classNames.contributorImage}
+              src={assigneeProfilePic}
+              alt="No contributor"
```

Comment:
Seems like the wrong alt text, especially if we have the contributor image show up

Reply:
shall I show up contributor name for alt?